salt/serializers/msgpack.py

```diff
@@ -46,7 +39,6 @@ def _serialize(obj, **options):
     def _deserialize(stream_or_string, **options):
         try:
             options.setdefault("use_list", True)
-            options.setdefault("raw", False)
```

Comment:
why are we removing the options here and on https://github.com/saltstack/salt/pull/58658/files#diff-0d5904efab02aab32413f541cbd67382L66 ? Different versions of msgpack will support different args.